edit-post/components/plugin-sidebar/style.scss

```diff
@@ -18,4 +18,8 @@
   }
 }
 
+.plugin-sidebar-error {
```

Comment:
I think this is a very basic support for plugin errors :)

@karmatosed @jasmussen @mtias  - do we have any plan how to handle errors for plugins? Should we provide a fallback for both `development` and `production`? Should the differ?

Reply:
Not sure, it would definitely be nice with a unified pattern, but that also has technical questions I'm not skilled enough to have insight in.

Reply:
I will share screens later :)